### PR TITLE
All users to specify extra URNs columns to include on the flow result…

### PR DESCRIPTION
**Pull request by norkans7**
…s export

### CODE REVIEW

**Comment by rowanseymour** [comment]
@norkans7 can you remind what the user was requesting here? As it is this change would let people generate exports with URNs duplicated in the URN column and in the scheme specific columns.

**Comment by norkans7** [comment]
for flow results the user can now specify more URNs to include extra contact URNs such as email.

Since the URN column show the URN used to interact with the flow I thought it is fine the extra column can be a duplicate

**Comment by rowanseymour** [comment]
Part of me hates that we have to keep adding more columns to the flow result exports... another part of me thinks we should embrace completely customizable exports where users can pick whatever columns they want from a list of URN schemes and contact fields.

File: temba/flows/tests.py [hunk 1]
```diff
@@ -992,32 +994,34 @@ def test_export_results(self):
         sheet_runs, sheet_contacts = workbook.worksheets
 
         self.assertEqual(len(list(sheet_contacts.rows)), 3)  # header + 2 contacts
-        self.assertEqual(len(list(sheet_contacts.columns)), 10)
+        self.assertEqual(len(list(sheet_contacts.columns)), 12)
 
-        self.assertExcelRow(sheet_contacts, 0, ["Contact UUID", "URN", "Name", "Groups", "Age",
+        self.assertExcelRow(sheet_contacts, 0, ["Contact UUID", "URN", "Name", "Groups", "Twitter", "Line", "Age",
```
Comment (rowanseymour) [inline temba/flows/tests.py]: Seems more natural if the extra URN columns are beside the preferred URN column

Comment (norkans7) [inline temba/flows/tests.py]: Okay

**Comment by norkans7** [comment]
@rowanseymour I think this is ready

**Comment by rowanseymour** [comment]
@norkans7 can you add a screenshot of the UI change

**Comment by norkans7** [comment]
export_flow_results_urns

**Pull request merged**